### PR TITLE
New icons

### PR DESCRIPTION
**Pull request by sroy3**
New main menu icons and help icon.

### CODE REVIEW

File: resources/icons/svg/help.svg [hunk 1]
```diff
@@ -1,5 +1,4 @@
-<svg width="20" height="20"  x="0px" y="0px" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg" xmlns:svg="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
-    <path fill-rule="evenodd" clip-rule="evenodd" d="m10,4.6c-2,0 -3.7,1.6 -3.7,3.7l1.8,0c0,-1 0.8,-1.8 1.8,-1.8s1.8,0.8 1.8,1.8c0,1.8 -2.8,1.6 -2.8,4.6l1.8,0c0,-2.1 2.8,-2.3 2.8,-4.6c0.2,-2.1 -1.5,-3.7 -3.5,-3.7z" id="svg_3"/>
-    <rect x="9.1" y="13.8" width="1.8" height="1.8" id="svg_4"/>
-    <path d="m10,17.2c-4,0 -7.2,-3.2 -7.2,-7.2s3.2,-7.2 7.2,-7.2s7.2,3.2 7.2,7.2s-3.2,7.2 -7.2,7.2zm0,-16.2c-5,0 -9,4 -9,9s4,9 9,9c5,0 9,-4 9,-9s-4,-9 -9,-9z" clip-rule="evenodd" fill-rule="evenodd" id="svg_5"/>
-</svg>
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="-948 530 22 22" enable-background="new -948 530 22 22">
+<path d="m-937 532c-5 0-9 4-9 9s4 9 9 9 9-4 9-9-4-9-9-9" fill="#f7f8f9"/>
```
Comment (GermainBergeron) [inline resources/icons/svg/help.svg]: The fill should probably be "currentColor" like the other icons

File: resources/icons/svg/help.svg [hunk 2]
```diff
@@ -1,5 +1,4 @@
-<svg width="20" height="20"  x="0px" y="0px" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg" xmlns:svg="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
-    <path fill-rule="evenodd" clip-rule="evenodd" d="m10,4.6c-2,0 -3.7,1.6 -3.7,3.7l1.8,0c0,-1 0.8,-1.8 1.8,-1.8s1.8,0.8 1.8,1.8c0,1.8 -2.8,1.6 -2.8,4.6l1.8,0c0,-2.1 2.8,-2.3 2.8,-4.6c0.2,-2.1 -1.5,-3.7 -3.5,-3.7z" id="svg_3"/>
-    <rect x="9.1" y="13.8" width="1.8" height="1.8" id="svg_4"/>
-    <path d="m10,17.2c-4,0 -7.2,-3.2 -7.2,-7.2s3.2,-7.2 7.2,-7.2s7.2,3.2 7.2,7.2s-3.2,7.2 -7.2,7.2zm0,-16.2c-5,0 -9,4 -9,9s4,9 9,9c5,0 9,-4 9,-9s-4,-9 -9,-9z" clip-rule="evenodd" fill-rule="evenodd" id="svg_5"/>
-</svg>
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="-948 530 22 22" enable-background="new -948 530 22 22">
```
Comment (GermainBergeron) [inline resources/icons/svg/help.svg]: The viewbox seems suspicious, maybe check with UX if they can clean it up

File: resources/icons/svg/menu-infrastructure.svg [hunk 3]
```diff
@@ -1,79 +1 @@
...
-<polygon fill="none" points="480,224 480,192 448,192 416,192 416,224 448,224 "/>
-<polygon points="416,32 96,32 0,128 512,128 "/>
-<path d="M0,384h512v-96H0V384z M416,320h64v32h-64V320z"/>
-<path d="M0,256h512v-96H0V256z M416,192h32h32v32h-32h-32V192z"/>
-<rect y="416" width="512" height="64"/>
-</svg>
+<svg viewBox="0 0 16 15" xmlns="http://www.w3.org/2000/svg"><g fill="currentColor" fill-rule="evenodd"><path d="m0 14.744c0 .141.102.256.251.256h15.497c.139 0 .251-.114.251-.256v-2.488c0-.141-.102-.256-.251-.256h-15.497c-.139 0-.251.114-.251.256v2.488m1-.744h14v-1h-14v1"/><path d="m0 10.744c0 .141.102.256.251.256h15.497c.139 0 .251-.114.251-.256v-2.488c0-.141-.102-.256-.251-.256h-15.497c-.139 0-.251.114-.251.256v2.488m13-.744h2v-1h-2v1m-12 0h11v-1h-11v1"/><path d="m0 6.744c0 .141.102.256.251.256h15.497c.139 0 .251-.114.251-.256v-2.488c0-.141-.102-.256-.251-.256h-15.497c-.139 0-.251.114-.251.256v2.488m13-1.744h1 1v1h-1-1v-1m-12 1h11v-1h-11v1"/><path d="m3.341 0c-.275 0-.664.161-.852.344l-2.384 2.312c-.196.19-.122.344.139.344h15.512c.273 0 .327-.161.139-.344l-2.385-2.312c-.196-.19-.57-.344-.852-.344h-9.317m-.07 1h9.458l1.031 1h-11.52l1.031-1"/></g></svg>
```
Comment (GermainBergeron) [inline resources/icons/svg/menu-infrastructure.svg]: This one as a viewbox of 16 by 15, it probably won't be easy to align with others

File: resources/icons/svg/help.svg (hunk 1 above)
Comment (sroy3) [inline resources/icons/svg/help.svg]: It is a gray background that should always be there. That is what the UX team said and that is what they gave me.

**Pull request merged**